ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -0,0 +1,144 @@
+	<xs:simpleType name="timingType">
+		<xs:union
+			memberTypes="ebuttdt:smpteTimingType ebuttdt:mediaTimingType ebuttdt:clockTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="durationTimingType">
+		<xs:union 
+			memberTypes="ebuttdt:clockTimingType ebuttdt:mediaTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="smpteTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-2][0-9]:[0-5][0-9]:[0-5][0-9]:[0-9][0-9]"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="clockTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-2][0-9]:[0-5][0-9]:[0-5][0-9]:[0-9][0-9]"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="mediaTimingType">
+		<xs:union memberTypes="ebuttdt:timecountTimingType ebuttdt:fullClockTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="timecountTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-9]+(\.[0-9]+)?(h|m|s|ms)"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="fullClockTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-9][0-9][0-9]*:[0-5][0-9]:[0-6][0-9](\.[0-9]+)?"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="lengthType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[+-]?\d*\.?\d+(px|c|%)"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="rgbHexColorType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="#[a-fA-F\d]{6}"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="rgbaHexColorType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="#[a-fA-F\d]{8}"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="rgbColorType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="rgb\(\d\d?\d?,\s?\d\d?\d?,\s?\d\d?\d?\)"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="rgbaColorType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="rgba\(\d\d?\d?,\s?\d\d?\d?,\s?\d\d?\d?,\s?\d\d?\d?\)"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="namedColorType">
+		<xs:restriction base="xs:token">
+			<xs:enumeration value="transparent"/>
+			<xs:enumeration value="black"/>
+			<xs:enumeration value="silver"/>
+			<xs:enumeration value="gray"/>
+			<xs:enumeration value="white"/>
+			<xs:enumeration value="maroon"/>
+			<xs:enumeration value="red"/>
+			<xs:enumeration value="purple"/>
+			<xs:enumeration value="fuchsia"/>
+			<xs:enumeration value="magenta"/>
+			<xs:enumeration value="green"/>
+			<xs:enumeration value="lime"/>
+			<xs:enumeration value="olive"/>
+			<xs:enumeration value="yellow"/>
+			<xs:enumeration value="navy"/>
+			<xs:enumeration value="blue"/>
+			<xs:enumeration value="teal"/>
+			<xs:enumeration value="aqua"/>
+			<xs:enumeration value="cyan"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="delayTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[1-9][0-9]*(ms|s|m|h)"></xs:pattern>
```

Comment:
Delays can be negative, and may have leading zeroes and fractions. So I think this should be more like:

`[+-]?[0-9]+(.[0-9]+)?(ms|s|m|h)` (which I haven't verified - not sure if the `.` needs to be escaped, for example)